baking/src/tezos_baking/tezos_setup_wizard.py

```diff
@@ -881,63 +881,57 @@ def get_current_cycle():
 
         current_cycle = get_current_cycle()
 
-        # TODO remove this check when ParisB protocol is activated at mainnet
-        if adaptive_issuance_launch_cycle is not None and int(current_cycle) >= int(
```

Comment:
Nitpick: we don't need to have `adaptive_issuance_launch_cycle` variable and `get_adaptive_issuance_launch_cycle` function anymore